src/Controller/UserController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\UserManager;
+
+class UserController extends AbstractController
+{
+    /**
+     * List users
+     */
+    public function index(): string
```

Comment:
Optionnal : We saw in the yesterday workshop how to add a Global variable in twig and how to hydrate it on construct with the user (which u will need in all the methods / pages of the controller / views)